OpenRA.Mods.Common/Widgets/TextNotificationsDisplayWidget.cs

```diff
@@ -18,7 +18,7 @@ namespace OpenRA.Mods.Common.Widgets
 {
 	public class TextNotificationsDisplayWidget : Widget
 	{
-		public readonly int RemoveTime = 0;
+		public readonly int Duration = 0;
```

Comment:
I'm not entirely convinced this is a great name either but it seems better than "RemoveTime".

Reply:
DisplayDuration?

Reply:
And after watching the Video recommend from @IceReaper in Discord - https://m.youtube.com/watch?v=-J3wNP6u5YU - we even can name ist DisplayDurationMilliseconds - if that won‘t use too much Bytes :-)

Reply:
Yes, making clear what's the unit of measurement is great. `DisplayDurationMilliseconds` is a bit lengthy but it's unambiguous and self-documenting. Maybe just `DurationMilliseconds`?

Reply:
You could abbreviate Milliseconds to Ms if the former is too long for your taste. We use `DisplayDuration` elsewhere, so either that or `DisplayDurationMs`/`DisplayDurationMilliseconds` would fit imo.